servatrice/src/serversocketinterface.cpp

```diff
@@ -984,13 +1027,22 @@ Response::ResponseCode AbstractServerSocketInterface::cmdActivateAccount(const C
 {
     QString userName = QString::fromStdString(cmd.user_name());
     QString token = QString::fromStdString(cmd.token());
+	QString clientID = "UNKNOWN"; //TODO:  ADD CLIENT ID INTO ACTIVATION PROTOCOL COMPONENTS FOR AUDIT PURPOSES
```

Comment:
I agree with you that it would be more useful for audit purposes to add the client_id to the activate account message.